config.yaml

```diff
@@ -33,9 +95,24 @@ weather:
             impulse_temp: 10 ## deg C
             impulse_duration: 60 ## seconds
             impulse_cycle: 600 ## seconds
+        thresholds:
+          sky-ambient:
+            cloudy: -25., -15.
+            very_cloudy: -15., .inf
+          wind_speed:
+            windy: 50., 75.
+            very_windy: 75., .inf
+          wind_gust:
```

Comment:
You need a 'not gusty' (or equivalent) category e.g. `0, 50`, otherwise `_get_status()` is going to mark non-gusty conditions with the 'invalid' default status.

Reply:
Included the 'calm' category (equivalent to 'not gusty').